src/advanced-settings/AdvancedSettings.jsx

```diff
@@ -244,8 +242,8 @@ const AdvancedSettings = ({ intl, courseId }) => {
         <AlertMessage
           show={saveSettingsPrompt}
           aria-hidden={saveSettingsPrompt}
-          aria-labelledby={intl.formatMessage(messages.alertWarningAriaLabelledby)}
-          aria-describedby={intl.formatMessage(messages.alertWarningAriaDescribedby)}
+          aria-labelledby="advancedSettingsAlertWarningTitle"
```

Comment:
```suggestion
          aria-labelledby="notification-warning-title"
```

Reply:
we used have
```js
  alertWarningAriaLabelledby: {
    id: 'course-authoring.advanced-settings.alert.warning.aria.labelledby',
    defaultMessage: 'notification-warning-title',
  },
```
and then we used that message in
```jsx
aria-labelledby={intl.formatMessage(messages.alertWarningAriaLabelledby)}
```
which means this evaluates to
```jsx
aria-labelledby="notification-warning-title"
```